v1/plugin/plugin.go

```diff
@@ -216,10 +216,7 @@ func (ts tlsServerDefaultSetup) makeTLSConfig() *tls.Config {
 	config := tls.Config{
 		ClientAuth:               tls.RequireAndVerifyClientCert,
 		PreferServerCipherSuites: true,
-		CipherSuites: []uint16{
-			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
-			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
-		},
+		MinVersion:               tls.VersionTLS12,
```

Comment:
isn't this a breaking change?
if you're sure to change this, probably we should update https://github.com/librato/snap/blob/290933aaf3e241a89636fc962b8209c8694442ef/docs/SECURE_PLUGIN_COMMUNICATION.md#tls-setup-requirements

Reply:
hm... you're right, will double-check that 👍

Reply:
related: https://github.com/golang/go/issues/45430

Reply:
well, gosec was right - both of the cipher suites I've removed are indeed blacklisted as insecure: https://golang.org/src/net/http/h2_bundle.go#L407 so we should not use those (and they may fail on negotiation anyway)

Also, I can't see why it might not work given the client part in snap: https://github.com/librato/snap/blob/d17832abf3330d66d1fccbc5a64e551fb2aa4cfd/grpc/common/credentials.go#L65